contracts/staking/Staking.sol

```diff
@@ -114,26 +114,9 @@ contract Staking is StakingV2Storage, GraphUpgradeable, IStaking, Multicall {
         bytes32 metadata
     );
 
-    /**
-     * @dev Emitted when `indexer` collected `tokens` amount in `epoch` for `allocationID`.
-     * These funds are related to `subgraphDeploymentID`.
-     * The `from` value is the sender of the collected funds.
-     */
-    event AllocationCollected(
-        address indexed indexer,
-        bytes32 indexed subgraphDeploymentID,
-        uint256 epoch,
-        uint256 tokens,
-        address indexed allocationID,
-        address from,
-        uint256 curationFees,
-        uint256 rebateFees
-    );
-
     /**
      * @dev Emitted when `indexer` close an allocation in `epoch` for `allocationID`.
      * An amount of `tokens` get unallocated from `subgraphDeploymentID`.
-     * The `effectiveAllocation` are the tokens allocated from creation to closing.
      * This event also emits the POI (proof of indexing) submitted by the indexer.
      * `isPublic` is true if the sender was someone other than the indexer.
      */
```

Comment:
Yes I intentionally wanted to break with the signature by changing the name of the event since conceptually they are different things. I'll check with Juan but seems like we need to update the subgraph to account for the new event, and the current handlers can be removed. I'll address that on a network subgraph PR.